.vscode/launch.json

```diff
@@ -7,9 +7,7 @@
 			"name": "Gulp Build",
 			"program": "${workspaceFolder}/node_modules/gulp/bin/gulp.js",
 			"stopOnEntry": true,
-			"args": [
```

Comment:
The change on this line and all the following in this file seem to be formatting changes only, and likely to cause unnecessary rebase conflicts.

There six other files in this PR that are similar - lots of formatting related changes. I feel like it would be easier to address this now but modifying the related commits or redoing them somehow - without formatting changes.

This may seem like a lot of work (using `git checkout <commit-hash> -- <file-path>` could help quite a bit) but I think it will save time in future.